### ISSUE TITLE
you are tracking product id's instead of product SKU's

### ISSUE THREAD

**Issue by dc006** [facebookincubator/facebook-for-magento#95]
Hi I've tested your Facebook_AdsExtension for magento 1
I've noticed that you are tracking product id's instead of product SKU's
I this intentional?

For example this is my Purchase Event on checkout thankyou page

```
CUSTOM PARAMETERS SENT
content_type: product
content_ids: 
["7446","7451","7452","2270","2258","2688","2692","2796","2664","4643","2644","7449","3006","2612","2344","7453"]
value: 120.76
source: magento
pluginVersion: 2.6.0
currency: GBP
version: 1.9.3.7
```

however my order have product SKU's ["TEST123","TESTFROZEN","TESTALCOHOL","2270","2258","2688","2692","2796","2664","3017c","2644","DC003","3001","2612","2344","241222"]

**Comment by dmitridr** [comment]
Hi,

Thanks for testing our extension.

Yes, we use the IDs because they are guaranteed to be unique. For SKU it's possible (unless I am very wrong) that two products have the same SKU, we'd like to avoid that situation when uploading products to FB since the ID sent to facebook must be unique or there will be an error.

The ids are only used to match in the catalog, so as long as the ids match (between pixel and catalog) it should not matter what they are. 

If you need SKU for other purposes, we can consider uploading SKU into an extra field which can be accessed on Facebook. Please let us know.

**Comment by chrisastley** [comment]
@dmitridr Magento can only have unique SKU's. It would be better to use these in case a product is accidentally deleted and then recreated.

**Comment by chrisastley** [comment]
Maybe it should be made with an admin option to select either ID (legacy) or sku to enable the merchant the option with no issues for existing or new users.

**Comment by dmitridr** [comment]
Yes, you could make it an advanced option. The problem is if you change it later you will have major problems... on your next feed upload all existing products will be deleted, and reuploaded as new products... so we would need to add a warning when changing the setting.
